includes/User.php

```diff
@@ -2174,6 +2174,8 @@ public function setPassword( $str ) {
 		}
 
 		$this->setInternalPassword( $str );
+		if($clearTokens)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)